src/main/java/frc/robot/commands/AimWithLimelight.java

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+/*
+ * Asimov's Laws:
+ * The First Law: A robot may not injure a human being or, through inaction, allow a human being to come to harm.
+ * The Second Law: A robot must obey the orders given it by human beings except where such orders would conflict with the First Law.
+ * The Third Law: A robot must protect its own existence as long as such protection does not conflict with the First or Second Law.
+ */
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.RobotContainer;
+import frc.robot.subsystems.Drive;
+import frc.robot.subsystems.Limelight;
+
+public class AimWithLimelight extends Command {
+  private Drive drive;
+  private Limelight limelight;
+
+  private boolean finished;
+  private int counter;
+
+  private double steerStrength,
+      distanceFromTarget,
+      mountHeight,
+      mountAngle,
+      driveStrength,
+      speedLimit,
+      turnDoneThreshold,
+      distanceDoneThreshold,
+      targetHeight;
+
+  /**
+   * @param limelight The limelight to aim with.
+   * @param steerStrength How hard to turn towards the target; between 0 and 1.
+   * @param distanceFromTarget How far from in inches we want to be from the target when we
+   *     dispense.
+   * @param mountHeight The height of the limelight off the floor.
+   * @param mountAngle The number of degrees the limelight is mounted back from perfectly vertical.
```

Comment:
Does positive mean it's mounted pointing down from perfectly vertical or pointing up from perfectly vertical?